.eslintrc

```diff
@@ -1,3 +1,4 @@
 {
   "extends": "airbnb/base",
+  "parser": "babel-eslint"
```

Comment:
The purpose of breaking down PRs is typically to make the PRs smaller so there's less things for reviewers to consider when reviewing each PR.

Additionally, it makes sure that backwards compatibility doesn't break for library/build upgrades.

However, I think I meant to comment on `plugins` in `Gruntfile` rather than the `.eslintrc` =/